lib/configgin.rb

```diff
@@ -66,6 +87,51 @@ def render_job_templates(jobs, job_configs)
     end
   end
 
+  # Some pods might have depended onthe properties exported by this pod; locate
```

Comment:
onthe -> on the.

Reply:
Fixed in da315a15408eec9730f51d8d295c513d19f22eb2